src/devices/SocketCan/CanFrame.cs

```diff
@@ -0,0 +1,145 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Sockets;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace Iot.Device.SocketCan
+{
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
Structs are always `LayoutKind.Sequential`.

Reply:
actually seems there is a tiny difference with and without attribute: no attribute generates warning for unused field such as `_padding`. I'll only use it when needed